concourse/steps/rm_pr_label.mako

```diff
@@ -25,6 +27,18 @@ with open(pr_id_path) as f:
   pr_id = int(f.read().strip())
 
 repository = github_api.repository('${main_repo.repo_owner()}', '${main_repo.repo_name()}')
+build_url = own_running_build_url()
+github_helper = GitHubRepositoryHelper(
+    owner='${main_repo.repo_owner()}',
+    name='${main_repo.repo_name()}',
+    github_cfg=github_cfg,
+)
+github_helper.add_comment_to_pr(
+    pull_request_number=pr_id,
+    comment=(
+        f"PR build URL: {build_url}"
```

Comment:
Is is rather short, especially compared to the other comments. Consider something like

> A build of this pull request has started. You can check on its progress here: <url>

or similar